packages/core/http/core-http-server/src/router/route.ts

```diff
@@ -120,6 +120,18 @@ export interface RouteConfigOptions<Method extends RouteMethod> {
    */
   xsrfRequired?: Method extends 'get' ? never : boolean;
 
+  /**
+   * Defines access permission for a route
```

Comment:
Maybe: "Intended request origin of the route". I think it is OK to call the property `access` but I think we should make it clear in the doc comment this is not a security feature.